crates/router/tests/connectors/utils.rs

```diff
@@ -133,8 +133,8 @@ pub trait ConnectorActions: Connector {
             self.get_auth_token(),
             enums::AuthenticationType::NoThreeDs,
             payment_data.unwrap_or_else(|| types::RefundsData {
-                amount: 100,
-                currency: enums::Currency::USD,
+                amount: 1000,
+                currency: enums::Currency::PLN,
```

Comment:
The currency being `PLN` might not be required for other connectors. Try to update it only in tests for this specific connector.